src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -27,7 +27,7 @@ export class GraphStyles {
     };
 
     const getTLSValue = (ele: any, tlsValue: string, nonTlsValue: string): string => {
-      if (ele.data('enabledmTLS') && ele.data('edgeLabelMode') === EdgeLabelMode.MTLS_ENABLED) {
+      if (ele.data('isMTLS') && ele.data('edgeLabelMode') === EdgeLabelMode.MTLS_ENABLED) {
```

Comment:
Again, maintain consistency across server and client side:
https://github.com/kiali/kiali/pull/292/files#diff-89df2dfb7238cfb78adc212f2a0581aeR64

enabledmTLS, isMTLS?

Reply:
OK, so iiuc you want consistency between the ui-side, json and server-side naming.  The json and ui-side already match.  I'll update the field names on the server-side.